src/apps/service/util.clj

```diff
@@ -11,8 +11,8 @@
   (partial sort-by
            (keyword sort-field)
            (if (and sort-dir (= (string/upper-case sort-dir) "DESC"))
-             #(compare (string/lower-case %2) (string/lower-case %1))
-             #(compare (string/lower-case %1) (string/lower-case %2)))))
+             #(compare (string/lower-case (str %2)) (string/lower-case (str %1)))
+             #(compare (string/lower-case (str %1)) (string/lower-case (str %2))))))
```

Comment:
This `app-sorter` function was throwing an NPE for fields with non-string values. This bug probably wasn't caught before now since the UI currently does not page app listings, and so it currently does all its own sorting in the browser.